src/countItems.js

```diff
@@ -12,11 +12,24 @@
  *  string in the input array
  */
 function countItems(array) {
-  // This is your job. :)
+  let count = 0
+  let dict = {};
+  for (let item of array) {
+    if (item in dict) {
```

Comment:
This isn't quite what you want. It will return `true` if `dict[item]` is defined, e.g., `'length' in []` is `true` because `[].length` is defined.

Use `hasOwnProperty` to check.